docker-image/Dockerfile.arm64

```diff
@@ -1,22 +1,24 @@
 ARG QEMU_IMAGE=calico/go-build:latest
 FROM ${QEMU_IMAGE} as qemu
 
-FROM arm64v8/alpine:3.8 as base
+FROM arm64v8/debian:9.8-slim as base
```

Comment:
If we're including a statically compiled version if `tini`, do we need to use a base image at all? Could we make this `from SCRATCH`?

Reply:
Yeah good question. Let me test it out. 

The only discrepancy then is for the alternate Dockerfile arch's which have the `qemu` stuff: 
```
ARG QEMU_IMAGE=calico/go-build:latest
FROM ${QEMU_IMAGE} as qemu

FROM ppc64le/debian:9.8-slim as base
MAINTAINER Shaun Crampton <shaun@tigera.io>

# ...
COPY --from=qemu /usr/bin/qemu-ppc64le-static /usr/bin/

FROM scratch
...
```

There's a `COPY` command that moves the `qemu` binary into the `base` image.

Reply:
Nope, no bueno. 

We still need the base, for the `chmod` from the looks of the image build error below: 
```
rm -rf docker-image/bin
mkdir -p docker-image/bin
cp bin/calico-typha-amd64 docker-image/bin/
docker build --pull -t calico/typha:latest-amd64 --build-arg QEMU_IMAGE=calico/go-build:v0.20 --file ./docker-image/Dockerfile.amd64 docker-image
Sending build context to Docker daemon  48.66MB
Step 1/11 : FROM scratch
 --->
Step 2/11 : MAINTAINER Shaun Crampton <shaun@tigera.io>
 ---> Using cache
 ---> 1ebd2ca76dc6
Step 3/11 : ENV TINI_VERSION v0.18.0
 ---> Using cache
 ---> 4f5a6a63288a
Step 4/11 : ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-amd64 /sbin/tini
Downloading [==================================================>]  870.5kB/870.5kB
 ---> Using cache
 ---> b90ab755f968
Step 5/11 : RUN chmod +x /sbin/tini
 ---> Running in 378612694c12
OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown
Makefile:256: recipe for target 'calico/typha' failed
make: *** [calico/typha] Error 1
```

Reply:
Going to wait for a response here. But other than this one remaining thread I think other points have been resolved.